config/subsidy-application-management/forms/ukraine/proposal/versions/20220509100012/form.ttl

```diff
@@ -0,0 +1,210 @@
+    sh:description "contact information";
+    sh:order 2;
+    sh:name "Contactgegevens contactpersoon" ;
+    form:help "Dit is de persoon die gecontacteerd wordt bij de opvolging van dit dossier." ;
+    sh:group fields:4558f2d6-1ba4-4211-beb9-5e3a4b65b975 .
+
+fields:386530d7-b4b4-41c9-8146-e2d8d629d77f a form:PropertyGroup;
+    mu:uuid "386530d7-b4b4-41c9-8146-e2d8d629d77f";
+    sh:description "payment";
+    sh:order 3;
+    sh:name "Betaling" ;
+    sh:group fields:4558f2d6-1ba4-4211-beb9-5e3a4b65b975 .
+
+fields:d8bbe7f4-56d3-4c01-9f98-bad3d5baf261 a form:PropertyGroup;
+    mu:uuid "d8bbe7f4-56d3-4c01-9f98-bad3d5baf261";
+    sh:description "Accountability";
+    sh:order 3;
+    sh:name "Verantwoording" ;
+    sh:group fields:4558f2d6-1ba4-4211-beb9-5e3a4b65b975 .
+
+
+##########################################################
+# Contact info
+##########################################################
+
+fields:e19e245a-c24b-4fb5-84b8-a7e2744ce9a0 a form:Field ;
+    mu:uuid "e19e245a-c24b-4fb5-84b8-a7e2744ce9a0";
+    sh:name "Voornaam contactpersoon" ;
+    sh:order 30 ;
+    sh:path ( schema:contactPoint foaf:firstName ) ;
+    form:validations
+    [ a form:RequiredConstraint ;
+        form:grouping form:Bag ;
+        sh:path ( schema:contactPoint foaf:firstName ) ;
+        sh:resultMessage "Dit veld is verplicht."@nl
+    ] ;
+    form:displayType displayTypes:defaultInput ;
+    sh:group fields:2232cd10-01d6-4f67-966a-2c22932437b4 .
+
+fields:82258a58-6c67-4b0a-a26d-0676b88886ad a form:Field ;
+    mu:uuid "82258a58-6c67-4b0a-a26d-0676b88886ad";
+    sh:name "Familienaam contactpersoon" ;
+    sh:order 31 ;
+    sh:path ( schema:contactPoint foaf:familyName ) ;
+    form:validations
+    [ a form:RequiredConstraint ;
+        form:grouping form:Bag ;
+        sh:path ( schema:contactPoint foaf:familyName ) ;
+        sh:resultMessage "Dit veld is verplicht."@nl
+    ] ;
+    form:displayType displayTypes:defaultInput ;
+    sh:group fields:2232cd10-01d6-4f67-966a-2c22932437b4 .
+
+fields:6285341d-a0ad-4933-bb5f-c49016b0f935 a form:Field ;
+    mu:uuid "6285341d-a0ad-4933-bb5f-c49016b0f935";
+    sh:name "Telefoonnummer" ;
+    sh:order 32 ;
+    sh:path ( schema:contactPoint schema:telephone ) ;
+    form:validations
+    [ a form:RequiredConstraint ;
+        form:grouping form:Bag ;
+        sh:path ( schema:contactPoint schema:telephone ) ;
+        sh:resultMessage "Dit veld is verplicht."@nl
+    ] ,
+    [ a form:ValidPhoneNumber ;
+        form:grouping form:MatchEvery ;
+        form:defaultCountry "BE" ;
+        sh:path ( schema:contactPoint schema:telephone ) ;
+        sh:resultMessage "Geef een geldig telefoonnummer in."@nl
+    ] ;
+    form:displayType displayTypes:defaultInput ;
+    sh:group fields:2232cd10-01d6-4f67-966a-2c22932437b4 .
+
+fields:31c743c7-b243-4ca7-a6c1-dc5f03cb5afb a form:Field ;
+    mu:uuid "31c743c7-b243-4ca7-a6c1-dc5f03cb5afb";
+    sh:name "Mailadres" ;
```

Comment:
```suggestion
    sh:name "E-mailadres" ;
```

I know that's how it's written in Miro, but this is the correct spelling so I would use that instead 😄.

Reply:
The Gemeenten & OCMW's has just been implemented

Reply:
Since this is new code I would simply do it correctly already. Anyways, a separate issue will do I guess.